pkg/cmd/backtest.go

```diff
@@ -295,7 +295,25 @@ var BacktestCmd = &cobra.Command{
 		}
 
 		backTestIntervals := []types.Interval{types.Interval1h, types.Interval1d}
-		exchangeSources, err := toExchangeSources(environ.Sessions(), startTime, endTime, backTestIntervals...)
+		// default interval is 1m for all exchanges
+		requiredInterval := types.Interval1m
+
+		allKLineIntervals := map[types.Interval]struct{}{}
+		for _, interval := range backTestIntervals {
+			allKLineIntervals[interval] = struct{}{}
+		}
+		for _, session := range environ.Sessions() {
+			for _, sub := range session.Subscriptions {
+				if sub.Channel == types.KLineChannel {
+					if sub.Options.Interval == types.Interval1s {
```

Comment:
add a comment for "// if any subscription is 1s, then we will use 1s for back-testing"